### PR TITLE
chore: run pre-commit on files

### PR DESCRIPTION
**Pull request by hackaugusto**
## Describe your changes

apply the fixers to the codebase (otherwise these changes get mixed in commits that are doing functional changes and makes it harder to review/track the changes)

## Checklist before requesting a review
- Repo forked and branch created from `next` according to naming convention.
- Commit messages and codestyle follow [conventions](./CONTRIBUTING.md).
- Relevant issues are linked in the PR description.
- Tests added for new functionality.
- Documentation/comments updated according to changes.

### CODE REVIEW

**Comment by bobbinth** [comment]
Overall, looks good - thank you for doing this! But let's merge this after #891 is merged.

**Comment by hackaugusto** [comment]
@bobbinth updated the PR

**Review by bobbinth — APPROVED** [review]
Looks good! Thank you!

**Pull request merged**
